packages/config/src/bundleConfig.ts

```diff
@@ -10,10 +10,20 @@ export type TypeScriptValidationOptions = {
   throwOnError?: boolean;
 };
 
+export type CustomSerializationPluginOption = {
+  path: string;
+  options?: object;
+};
```

Comment:
Should we match how Babel, ESLint, Jest and others specify plugins?
```suggestion
export type CustomSerializationPluginOption =
  | string
  | [string, Record<string, unknown>];
```